src/main/java/com/powsybl/openloadflow/ac/outerloop/AbstractTransformerVoltageControlOuterLoop.java

```diff
@@ -28,8 +25,12 @@ public abstract class AbstractTransformerVoltageControlOuterLoop implements AcOu
     private static final double MIN_TARGET_DEADBAND_KV = 0.1; // kV
 
     protected static List<LfBranch> getControllerBranches(LfNetwork network) {
-        return network.getBranches()
-                .stream().filter(branch -> !branch.isDisabled() && branch.isVoltageController())
+        return network.getBuses().stream()
+                .filter(bus -> bus.isTransformerVoltageControlled())
+                .filter(bus -> bus.getTransformerVoltageControl().get().getMergeStatus() == VoltageControl.MergeStatus.MAIN // FIXME: is MAIN status needed as not hidden
```

Comment:
@geofjamg I think `MAIN` check is not needed as the voltage control is not hidden.